### ISSUE TITLE
{Feature}: Low Contrast: "GO TO PROJECT" Links, UI, small issue

### ISSUE THREAD

**Issue by AVbiser** [zetkin/app.zetkin.org#2698]
## Description:
Low Contrast: "GO TO PROJECT" Links

The red text (“GO TO PROJECT”) on a light grey background may not meet the WCAG 2.1 minimum contrast ratio of 4.5:1 (Contrast Ratio is 4.28:1). Users with low vision or color blindness might struggle to read these links.

## Steps to reproduce

**Be detailed, volunteers need to know exactly how to reproduce!**

1. Go to: https://app.dev.zetkin.org/organize/1/projects
2. scroll down to All projects
3. all GO TO PROJECT links on each individual project

### Expected Behaviour
Instead of the the red text “GO TO PROJECT” being #ED1C55 color, it can be changed to, for example, #E2124A color in order to meet WCAG color contrast guidlines.
<img width="597" alt="Image" src="https://github.com/user-attachments/assets/c4210114-ca01-40e1-8abb-3713981d73e2" />

### Actual Behaviour
<img width="598" alt="Image" src="https://github.com/user-attachments/assets/5eea15a8-315c-489c-9c0c-4c7618556cf6" />

**Comment by Bagera** [comment]
I agree that there should be better contrast here, or larger type, but I think we will solve later down the road.
As we are moving toward implementing our own design system (ZUI), these colors will change and the issue will be solved.
